### PR TITLE
Slime grinders suck less

### LINKED ISSUE #16939: Slime processor taking in slimes when in use
Processor shouldn't be taking them in while it's processing the previous ones. And after it's done it's saying processor is empty. I'm not sure if the slimes just disappeared or if it processed them as well. Anyways, should be an easy fix to add a check for the processor being in use.

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
tweak: Slime grinders now cease suction of new slimes while grinding
/:cl:

Fixes #16939

### CODE REVIEW

**Comment by bgobandit** [comment]
>not titling this "Slime processors suck less"

**Comment by coiax** [comment]
@bgobandit Done.

File: code/modules/food&drinks/kitchen machinery/processor.dm [hunk 1]
```diff
@@ -31,6 +31,10 @@
 
 /obj/machinery/processor/process()
 	..()
+	// The irony
+	// To be clear, if it's grinding, then it can't suck them up
```
Comment (ChangelingRain) [inline code/modules/food&drinks/kitchen machinery/processor.dm]: no space between the //and text

Comment (coiax) [inline code/modules/food&drinks/kitchen machinery/processor.dm]: I, uh, have been using that comment style everywhere. Is it a problem?

Comment (KazeEspada) [inline code/modules/food&drinks/kitchen machinery/processor.dm]: Leave him alone, its a comment.

**Pull request merged**